fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ImportPartitionMapper.java

```diff
@@ -130,128 +137,137 @@
+            return fhirDataSources;
+        } else {
+            logger.finer("getFhirDataSourcesForObjectStore: Succeed get BucketName!");
+        }
+        cosBucketName = cosBucketName.toLowerCase();
+        if (!cosClient.doesBucketExistV2(cosBucketName)) {
+            logger.warning("getFhirDataSourcesForObjectStore: Bucket '" + cosBucketName + "' not found!");
+            BulkDataUtils.listBuckets(cosClient);
+            return fhirDataSources;
+        }
+
+        ListObjectsV2Result result = null;
+        do {
+            if (result != null) {
+                nextToken = result.getNextContinuationToken();
+            }
+            ListObjectsV2Request request =
+                    new ListObjectsV2Request().withBucketName(cosBucketName).withMaxKeys(1000)
+                            .withContinuationToken(nextToken);
+            result = cosClient.listObjectsV2(request);
+            for (S3ObjectSummary objectSummary : result.getObjectSummaries()) {
+                boolean isToBeProccessed = false;
+                if (dsDataLocationInfo != null && !dsDataLocationInfo.trim().isEmpty()) {
+                    if (objectSummary.getKey().startsWith(dsDataLocationInfo.trim())) {
+                        isToBeProccessed = true;
+                    }
+                } else {
+                    isToBeProccessed = true;
+                }
+                if (isToBeProccessed) {
+                    logger.info("getFhirDataSourcesForObjectStore: ObjectStorge Object(" + objectSummary.getKey()
+                            + ") - " + objectSummary.getSize() + " bytes.");
+                    if (objectSummary.getSize() > 0) {
+                        fhirDataSources.add(new FhirDataSource(dsTypeInfo, objectSummary.getKey()));
+                    }
+                }
+            }
+        } while (result != null && result.isTruncated());
+
+        return fhirDataSources;
+    }
+
+    private List<FhirDataSource> getFhirDataSources(JsonArray dataSourceArray, BulkImportDataSourceStorageType type)
+            throws Exception {
+        List<FhirDataSource> fhirDataSources = new ArrayList<>();
+        for (JsonValue jsonValue : dataSourceArray) {
+            JsonObject dataSourceInfo = jsonValue.asJsonObject();
+            String dsTypeInfo = dataSourceInfo.getString(Constants.IMPORT_INPUT_RESOURCE_TYPE);
+            String dsDataLocationInfo = dataSourceInfo.getString(Constants.IMPORT_INPUT_RESOURCE_URL);
+
+            switch (type) {
+            case HTTPS:
+            case FILE:
+                fhirDataSources.add(new FhirDataSource(dsTypeInfo, dsDataLocationInfo));
+                break;
+            case AWSS3:
+            case IBMCOS:
+                fhirDataSources.addAll(getFhirDataSourcesForObjectStore(dsTypeInfo, dsDataLocationInfo));
+                break;
+            default:
+                break;
+            }
+        }
+
+        return fhirDataSources;
+    }
```

Comment:
not sure if any change made to the 2 above function ...

Reply:
made camelcase, and originally I had moved base64 processing around.  

Should be ok- now, no changes, just formatting and names